OctAssignments/MyUnderstandInHTML.html

```diff
@@ -1,30 +1,32 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
     <head>
+        <meta charset="utf-8"/>
         <title>
             My Understanding in Git
         </title>
     </head>
 
     <body>
-        <h>
+        <div>
             <b> 
-                <big> 
```

Comment:
Please read the message I posted in slack and update the file with good usage of tags. If any help is needed please ask in forums channel.

Reply:
I replaced <big> with  <span style="font-size:1.2em">, Is this valid?